setup_env_ds.sh

```diff
@@ -72,3 +77,6 @@ patch -sp0 < libgdiam-1.0.3.patch
 cd libgdiam-1.0.3
 mkdir build && cd build && cmake .. && make
 cd ../../../..
+patch -u algorithms/PCGCv2/test.py test.patch
```

Comment:
What is the patch for?

Reply:
remove colors from input point cloud, or the program (test.py) cannot be run

Reply:
Updated